03_it_analysis/it_analysis_data_prep_config.yaml

```diff
@@ -3,42 +3,70 @@ it_analysis_data_prep.py:
     srcs: 
         - 'discharge'
         - 'specific_conductance'
+        - 'prcp'
+        - 'tmax'
+        - 'tmin'
+        - 'wspdir'
```

Comment:
the variable `wspdir` incorporates wind speed and direction, it is calculated as -1*speed*cos(direction), so that wind from the North < 0 and wind from the south is > 0. I'll add this as a comment